support/oidc-discovery-provider/config.go

```diff
@@ -148,32 +173,17 @@ func ParseConfig(hclConfig string) (_ *Config, err error) {
 	}
 
 	switch {
```

Comment:
If this switch now has cases all where ACME != nil, we might as well just put it in an `if c.ACME != nil` block?